sites/all/modules/custom/debugacademy/debugacademy.module

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * Implements hook_menu().
+ */
+ 
+function debugacademy_menu() {
+	$items = array();
```

Comment:
@malikkotob by drupal coding standards, indentations should be 2 spaces, not tabs